Adding command to export to `index.html` (#1)
LiveDuo/destack-starter: README.md

```

Deploy it to the cloud with [Vercel](https://vercel.com/new) ([Documentation](https://nextjs.org/docs/deployment)).

> If you need to deploy it with `index.html`, export it with:

Reviewer: Export the project to static HTML with:

> If you need to deploy it with `index.html`, export it with:

Reviewer: ```suggestion
Export the project to static HTML with:
```

```sh
npm run export
```

> `index.html` will be exported in the `out` directory

Reviewer: ```suggestion
This will export `index.html` to the `out` directory.
```
